### PR TITLE
Multiple thinpools

### PR DESCRIPTION
**Pull request by fepitre**
Hello,
I've coded a first version for allowing automatic partitioning requests having multiple lvm thin pools. We are facing this configuration in Qubes. I've tried to be not too intrusive with this feature and if you are interested by those pieces of codes, remarks are very welcomed.

### CODE REVIEW

File: pyanaconda/modules/storage/partitioning/automatic/utils.py [hunk 1]
```diff
@@ -369,20 +369,35 @@ def schedule_volumes(storage, devices, scheme, requests, encrypted=False):
...
+        # make sure VG reserves space for the pool to grow if needed
+        container.thpool_reserve = DEFAULT_THPOOL_RESERVE
+
+    # TODO: Ensure requests are properly ordered:
+    #  VG -> Static LV
+    #  VG -> Thin Pool - > Thin Volume
     for request in requests:
         btr = bool(scheme == AUTOPART_TYPE_BTRFS and request.btr)
         lv = bool(scheme in (AUTOPART_TYPE_LVM, AUTOPART_TYPE_LVM_THINP) and request.lv)
+        thinlp = bool(scheme == AUTOPART_TYPE_LVM_THINP and request.lv and request.thin_pool)
```
Comment (jkonecny12) [inline pyanaconda/modules/storage/partitioning/automatic/utils.py:398]: I guess this should be just `thinp` as thin pool.

**Comment by jkonecny12** [comment]
Jenkins, it's ok to test.

File: pyanaconda/modules/storage/partitioning/automatic/utils.py (hunk 1 above)
Comment (fepitre) [reply pyanaconda/modules/storage/partitioning/automatic/utils.py:398]: Sure. Would you like me to fix the name right now or we wait for Jenkins feedback?

**Comment by fepitre** [comment]
I'm closing it as it would need to be reworked differently now.

**Pull request closed without merging**